action.yml

```diff
@@ -1,100 +1,104 @@
-name: 'Checkout'

```

Comment:
I don't know why this shows this big diff, I thought it could be formatting but it doesnt seem like it.

Reply:
If you want to see the reason, I would suggest to run `git show --word-diff --word-diff-regex=. 2bd5ef5487b953b5db186c7a68dcab4e4a8fe3fc -- action.yml` - you will see that it had to do with the line endings (maybe something like e.g. https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings can be of help?)